test/direnv-test-common.sh

```diff
@@ -92,6 +92,8 @@ test_start base
   direnv_eval
   echo "${HELLO}"
   test -z "${HELLO}"
+
+  unset WATCHES
```

Comment:
```suggestion
  unset DIRENV_WATCHES
```
Is that what you meant?

Reply:
Nope, we set `WATCHES` on line 77:

```
WATCHES=$DIRENV_WATCHES
```

Meaning that we need to unset it at the end of this test I think. Sorry I've not had much time to look at this PR